js/samples/04.q.questBot/.gitignore

```diff
@@ -113,6 +113,3 @@ env/.env.*.user
 env/.env.local
 appPackage/build
 .deployment
-
-.env.*
```

Comment:
Wouldn't we want to keep these, since we don't want these files to get checked in?

Reply:
we should only exclude .env not .env.*